tools/mmp/linker/MonoMac.Tuner/MonoMacNamespaces.cs

```diff
@@ -113,6 +114,122 @@ public void Process (LinkContext context)
 					// no sample
 					remove_dlopen = !namespaces.Contains (Namespaces.GameKit);
 					break;
+				case Constants.GameControllerLibrary:
```

Comment:
This duplication makes me :( . I was tempted to write reflection to covert the case enum to the name of the constant and read that in, but that seemed worse, kinda.

Thoughts?